src/index.jsx

```diff
@@ -69,22 +56,41 @@ function configure() {
   };
 }
 
-apiClient.ensurePublicOrAuthenticationAndCookies(
-  window.location.pathname,
-  (accessToken) => {
-    const { store, history } = configure();
+function initialize(accessToken) {
+  const { store, history } = configure();
+
+  if (accessToken) {
+    ReactDOM.render(<App store={store} history={history} />, document.getElementById('root'));
+    identifyAuthenticatedUser(accessToken.userId);
+    sendPageEvent();
+  } else {
+    // This should never happen, but it does sometimes.
+    // Add logging to learn more.
+    throw new Error('Empty accessToken returned from ensurePublicOrAuthenticationAndCookies callback.');
+  }
+}
+
+try {
+  if (configuration.ENVIRONMENT !== 'production') {
+    import(/* webpackChunkName: "react-axe" */ 'react-axe').then(({ default: axe }) =>
+      axe(React, ReactDOM, 1000));
+  }
 
-    if (accessToken) {
-      ReactDOM.render(<App store={store} history={history} />, document.getElementById('root'));
-      identifyAuthenticatedUser(accessToken.userId);
-      sendPageEvent();
-    } else {
-      // This should never happen, but it does sometimes.
-      // Add logging to learn more.
-      ReactDOM.render(<ErrorPage />, document.getElementById('root'));
-      NewRelicLoggingService.logError('Empty accessToken returned from ' +
-        'ensurePublicOrAuthenticationAndCookies callback.');
-      identifyAnonymousUser();
```

Comment:
This line to identify an anonymous user is intentionally removed.  We don't want to do that here - this is an error condition, not a normal thing for anonymous users.